librz/core/cmd/cmd_print.c

```diff
@@ -7288,4 +7224,58 @@ RZ_IPI RzCmdStatus rz_cmd_base64_decode_handler(RzCore *core, int argc, const ch
 	rz_cons_println((const char *)buf);
 	free(buf);
 	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_bitstream_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	st32 len = (st32)rz_num_math(core->num, argv[1]), skip = 0;
+	if (!rz_str_isnumber(argv[1]) || len < 0) {
+		RZ_LOG_ERROR("len is not a positive number\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	if (argv[2]) {
+		skip = (st32)rz_num_math(core->num, argv[2]);
+		if (!rz_str_isnumber(argv[2]) || skip < 0) {
+			RZ_LOG_ERROR("skip is not a positive number\n");
+			return RZ_CMD_STATUS_ERROR;
+		}
+	}
+	// `pb len skip` means skip <skip> bits then print <len> bits
+	char *buf = calloc(1, len + skip + 1);
+	if (!buf) {
+		RZ_LOG_ERROR("Fail to allocate memory\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_str_bits(buf, core->block, len + skip, NULL);
+	if (!RZ_STR_ISEMPTY(buf + skip)) {
+		rz_cons_println(buf + skip);
+	}
+	free(buf);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_byte_bitstream_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	if (!rz_str_isnumber(argv[1])) {
+		RZ_LOG_ERROR("len is not a number\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	ut64 cur_off = core->offset;
+	int len = (st32)rz_num_math(core->num, argv[1]);
+	if (len < 0) {
+		len *= -1;
+		rz_core_seek(core, cur_off - len, SEEK_SET);
+		rz_core_block_read(core);
+	}
```

Comment:
no need for this, remove

Reply:
Do you mean remove lines 7261~7267, which support commands like `pB -1`? This may break the test `NAME=pB -10`

Reply:
Do you mean remove lines 7261~7267?